lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -240,7 +241,7 @@ def _setup_mock_request(self, mock_request, include_depth=False):
             data["depth"] = 0
         self._set_mock_request_data(mock_request, data)
 
-    def create_thread_helper(self, mock_request, extra_data=None):
+    def create_thread_helper(self, mock_request, extra_data=None, extra_expected_data=None):
```

Comment:
One of these is the extra data we'd like to pass along with the request. The other is the extra expected data we expect back from the server.